devices/pine64-pinephone/modem.nix

```diff
@@ -1,57 +1,13 @@
 { config, lib, pkgs, ... }:
 
-# This, by default, boots the phone with the modem powered down.
-# In addition, the systemd service does not, by default, start the modem.
 {
+  systemd.packages = [
+    pkgs.eg25-manager
+  ];
 
-  # Shut down the modem in early boot
-  mobile.quirks.u-boot.additionalCommands = ''
-    # Properly shut off EG25 by pulling up PWRKEY.
-    gpio set 35
-    sleep 1
-    gpio clear 35
-  '';
```

Comment:
This would become more of a discussion for Tow-Boot...

... but should the modem be on during early boot or not?

I'm thinking about situations where you'd start a system that does not or cannot control the modem, then it is a needless power hog. But conversely, I'm thinking about the situation where you want the modem to be ready ASAP, starting it ASAP helps.

I guess the only correct answer is: make this a platform firmware setting.

Reply:
+1 to this being a platform firmware setting.

For now, I'm happy to keep this in mobile-nixos. I've updated this PR to not change `mobile.quirks.u-boot.additionalCommands`.